setup/sql/dbupdate_05.php

```diff
@@ -759,3 +759,28 @@
 	$ilDB->dropTableColumn('post_conditions', 'condition_type');
 }
 ?>
+
+<#5484>
+<?php
+include_once('./Services/Migration/DBUpdate_3560/classes/class.ilDBUpdateNewObjectType.php');
+
+$lp_type_id = ilDBUpdateNewObjectType::getObjectTypeId('lso');
+if ($lp_type_id) {
+	$ops_id = ilDBUpdateNewObjectType::getCustomRBACOperationId("lp_other_users");
+	ilDBUpdateNewObjectType::deleteRBACOperation($lp_type_id, $ops_id);
+	$ilDB->manipulate("DELETE FROM rbac_operations WHERE ops_id = ".$ilDB->quote($ops_id, "integer"));
```

Comment:
I do not think we want to do this. We did not add that permission in the first place, lets not remove it here.

Reply:
Please have a look at update step 5480. There i put the permission lp_other_user for lso into rbac system. So why not delete it at this place?